src/app/utilities/Toaster.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { expect } from 'chai';
+import { ToastyService, ToastOptions, ToastData, ToastyConfig } from 'ng2-toasty';
+import { Toaster } from './Toaster';
+
+describe('Toaster', () => {
+    let mockedToastyService: ToastyService;
+    let sut: Toaster;
+    let passedOptions: ToastOptions | string | number;
+
+    beforeEach(() => {
+        mockedToastyService = new ToastyService(new ToastyConfig());
+        sinon.stub(mockedToastyService, 'default')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'info')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'success')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'wait')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'warning')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'error')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
```

Comment:
Do you mean the `options: ToastOptions | string | number` notation?
if so, then it is simply the declaration of the variable options and its accepted types.

We usually see the standard:
`let myVar: boolean`
But you could make it so this variable accepts more than just a boolean value:
`let myVar: boolean | number`

in the second scenario you could assign an boolean value OR a number to this variable.